### DISCUSSION TITLE
Can this work with twitch?

### DISCUSSION THREAD

**Discussion by MichaelEpicA** [Voltstro-Studios/UnityWebBrowser#108] [General]
I've been searching everywhere for a browser that can work with twitch, and so far all of them I've tried have given me that error #4000 error. If not, can someone help me find a browser that works with twitch, or maybe get a way for it to work on twitch?

**Comment by Voltstro** [comment]
No UWB doesn't support Twitch, as Twitch uses proprietary codecs for it videos. See [this comment](https://github.com/Voltstro-Studios/UnityWebBrowser/discussions/79#discussioncomment-2758713) for more info on why proprietary codecs aren't supported.

I doubt you will find a browser for Unity that does ship with proprietary codecs, as no-one wants to deal with compiling CEF and the licenses for the codecs.

**Reply by MichaelEpicA** [reply]
Welp, that makes a LOT of sense. If I compile it into CEF, I'm guessing if the people actually notice my game without the proper licenses, legal trouble? (What codecs does twitch even use)

**Reply by Voltstro** [reply]
Yea, it would most likely just lead to a lot of legal trouble if the license holders do notice your game, but I wouldn't risk trying to get away using the codecs without a license.

Twitch appears to be using h.264.
